README.md

```diff
@@ -1 +1,51 @@
 # Logz.io Logs for Salesforce Commerce Cloud
+
+A Docker image purpose-built to monitor Salesforce Commerce Cloud (fka Demandware) logs
+
+## Usage
+
+### 1. Pull the Docker image
+
+Use the following docker command to run your container
+
+Donwload the logzio/webdav-fetcher image:
+
+```sh
+docker pull logzio/webdav-fetcher:latest
+```
+
+### 2. Run the container
+
+Use the following docker command to run your container.
+
+```sh
+docker run -d -e LOGZIO_SHIPPING_KEY=<logzio_shipping_token>  \
```

Comment:
Please rearrange the environment variables so Logzio variables will be next to each other . Also please change the environment variable to LOGZIO_SHIPPING_TOKEN